<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCustomStackedWidget" name="myStackedWidget">
      <widget class="QWidget" name="page">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 0, 127);</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>90</y>
          <width>171</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>PAGE 1</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 255, 255);</string>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>100</y>
          <width>241</width>
          <height>191</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #000;</string>
        </property>
        <property name="text">
         <string>PAGE 2</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(25, 25, 25);</string>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>90</y>
          <width>211</width>
          <height>151</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>PAGE 3</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="nxt">
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="page1">
         <property name="text">
          <string>Page 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="page2">
         <property name="text">
          <string>Page 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="prev">
         <property name="text">
          <string>Previous</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header location="global">Custom_Widgets.Widgets.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
